pandas/src/parser/tokenizer.h

```diff
@@ -145,30 +145,30 @@ typedef struct parser_t {
     io_callback cb_io;
     io_cleanup cb_cleanup;
 
-    int chunksize;  // Number of bytes to prepare for each chunk
+    int64_t chunksize;  // Number of bytes to prepare for each chunk
     char *data;     // pointer to data to be processed
-    int datalen;    // amount of data available
-    int datapos;
+    int64_t datalen;    // amount of data available
+    int64_t datapos;
 
     // where to write out tokenized data
     char *stream;
-    int stream_len;
-    int stream_cap;
+    int64_t stream_len;
+    int64_t stream_cap;
 
     // Store words in (potentially ragged) matrix for now, hmm
     char **words;
-    int *word_starts; // where we are in the stream
-    int words_len;
-    int words_cap;
+    int64_t *word_starts; // where we are in the stream
+    int64_t words_len;
+    int64_t words_cap;
 
     char *pword_start;    // pointer to stream start of current field
-    int word_start;       // position start of current field
+    int64_t word_start;       // position start of current field
 
-    int *line_start;      // position in words for start of line
-    int *line_fields;     // Number of fields in each line
-    int lines;            // Number of (good) lines observed
-    int file_lines;       // Number of file lines observed (including bad or skipped)
-    int lines_cap;        // Vector capacity
+    int64_t *line_start;      // position in words for start of line
+    int64_t *line_fields;     // Number of fields in each line
+    int64_t lines;            // Number of (good) lines observed
```

Comment:
Is int64_t really necessary here? 
Of course using smaller int won't save a lot of RAM, but it can be used just for consistency.

Reply:
Yes. Issue before with int was that we could only read 2147483647 rows which was limit of signed int32, at which point it overflowed to negative.